lms/djangoapps/instructor_task/tasks_helper.py

```diff
@@ -820,6 +820,17 @@ def upload_grades_csv(_xmodule_instance_args, _entry_id, course_id, _task_input,
             course_grade.letter_grade,
             student.id in whitelisted_user_ids
         )
+        if certificate_info[0] == 'Y':
+            # certificate_info[0] is eligible_for_certificate
+            TASK_LOG.info("Student is marked eligible_for_certificate in grade report "
```

Comment:
As per the last comment of @adampalay, Pls track the following in the log.
- What the grade percentage was 
- What the minimum passing grade cutoff is